src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -135,6 +138,9 @@ sealed internal class SqlInternalConnectionTds : SqlInternalConnection, IDisposa
         SqlFedAuthToken _fedAuthToken = null;
         internal byte[] _accessTokenInBytes;
 
+        // Kerberos ticket provided by OnRetrieveKerberosTicket
+        internal byte[] _kerberosTicketInBytes;
```

Comment:
I realise that this follow the naming convention from the line above, the same logic will apply to that field as well if someone added it today. 
Do we need the `InBytes` suffix? Do we have any other form of the same data (string etc) that is used? If not then it could just be called `_kerberosTicket`

Reply:
Agree with the comment. As you've noticed correctly the idea was to follow the established convention. Team may consider name refactoring.